### DISCUSSION TITLE
'.\ble-server-windows\ble_server.exe' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by BreweRail** [Novakasa/brickrail#171] [Q&A] [answered]
What is going wrong? I'm new at this. Did a new installation and get this error: '.\ble-server-windows\ble_server.exe' is not recognized as an internal or external command, operable program or batch file. Button 'Connect BLE Server' is not werking.

**Comment by Novakasa** [comment]
Can you explain in more detail  how you started brickrail?
Are you on windows 10? Windows 11?
Did you download the  `brickrail-windows-v1.0.0-alpha.5.zip` release from https://github.com/Novakasa/brickrail/releases/tag/v1.0.0-alpha.5 ?
Did you extract the .zip file before starting the Brickrail.exe?

**Reply by Novakasa** [reply, marked as answer]
solved in https://github.com/Novakasa/brickrail/issues/170
